### DISCUSSION TITLE
Plan for next release? 0.5.3, 0.6.0

### DISCUSSION THREAD

**Discussion by li-boxuan** [JanusGraph/janusgraph#2295] [Q&A] [answered]
Hi folks,

Is there any plan for 0.5.3 & 0.6.0 release? Are there any blockers for the releases?

@porunov what do you think? I am not sure but If I remember correctly, you are the release manager, right?

**Comment by porunov** [comment, marked as answer]
Hi @li-boxuan ,
We never discussed 0.5.3 and we didn't create a milestone  for it. I think we can just create 0.5.3 milestone, move necessary issues from 0.6.0 release and check if there are any outstanding PRs targeting this release. If there are no any PRs or plans for 0.5.3 release, I can go ahead and and start the release process (unless anyone else wants to do that).
As for 0.6.0, we have some outstanding PRs targeting the release. We need to try to review them and / or check if authors of those PRs are willing to resolve the requested changes. In case some PRs are stuck without any comments from an author, we can always retarget a PR to a next release. I would expect 0.6.0 version to be released somewhere at the beginning of 2021 (probably mid / late January).

**Reply by li-boxuan** [reply]
Sounds awesome! Having a 0.5.3 release without waiting for 0.6.0 would be very nice.

**Reply by li-boxuan** [reply]
The only outstanding PR targeting v0.5 that I found is your recent one: https://github.com/JanusGraph/janusgraph/pull/2290

**Reply by porunov** [reply]
Just created a 0.5.3 milestone. We will need to go through all closed PRs in 0.6.0 release and retarget those which are merged in v0.5 branch to 0.5.3 milestone

**Reply by li-boxuan** [reply]
Thanks. I just went through all commits on the v0.5 branch and added some missing ones to the 0.5.3 milestone.

**Reply by porunov** [reply]
Thanks! I pushed changes to #2290

**Reply by porunov** [reply]
v0.5.3 PR is opened here: #2301

**Reply by porunov** [reply]
v0.5.3 vote: https://groups.google.com/g/janusgraph-dev/c/MV0rgc4fXD4
